portfolio/src/main/webapp/pets.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Leo's  Pets</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div>
+        <img src="/images/pookie.JPG" alt="Pookie, Leo's dog"></img>
```

Comment:
`<img>` is what's called a "void" element in html, meaning that it has no body, so it doesn't need a closing tag.

Optionally, you can format it like an xml "self-closing tag" by writing `<img src="..." />` (note the `/` at the end of the "opening" tag, but that's not required in HTML).

Reply:
Makes sense, I'll remove the closing tag